src/PhpWord/Style/Chart.php

```diff
@@ -551,4 +579,68 @@ public function setShowGridX($value = true)
 
         return $this;
     }
+
+    /**
+     * @return null|float|int
+     */
+    public function minY()
+    {
+        return $this->minY;
+    }
+
+    /**
+     * @param null|float|int $minY
+     */
+    public function setMinY($minY): void
+    {
+        $this->minY = $minY;
+    }
```

Comment:
```suggestion
    public function setMinY($minY): self
    {
        $this->minY = $minY;
        
        return $this;
    }
```

@jfoucher Can you return `$this` for each setter, please ?